api/client/samples/prevented_plant/prevent-plant-model-overview.md

```diff
@@ -0,0 +1,51 @@
+<p align="center"><img width=8% src="https://gro-intelligence.com/images/logo.jpg"></p>
+
+# Prevent Plant Model for US Corn and Soybeans Using Gro
+
+## Goal & Process
+Basic Model Description: The prevent plant model endeavors to forecast prevent plant area for corn and soybeans at the county level. Using datasets in Gro, the model will examine the overall wetness across all Corn and Soybean Belt counties using soil moisture, rainfall, planting progress reports, and historical prevented plant area. 
+
+Prevented plant is the area farmers intended to plant, but were unable to, typically because of extreme weather. Farmers are entitled to a payout from revenue-insurance contracts if the ground remains unplanted (for more information see https://www.rma.usda.gov/Topics/Prevented-Planting)
+
+## Model Inputs
+__Corn Prevent Plant Model Inputs:__ 
+[SMOS Soil Moisture](https://app.gro-intelligence.com/dictionary/sources/43)

```

Comment:
SMOS is appearing on the same line as "Corn Prevent Plant Model Inputs" in the rendered page. Looks like a bug. Supposed to be bulleted?